index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kip's portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header-container">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <nav class="header-nav-lg">
+            <img src="images/ic_menu.jpg" alt="image">
+        </nav>>
+      <section class="about-container">
+        <div class="header-my-name">
+            <h1>Kiprono Japhet</h1>
+        </div>
+        <div class="header-about">
+            <p class="aboutme">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                  If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+        </div>
+        
+        <button>Start collaboration</button>
```

Comment:
- [ ] Please Implement the button interactions (enable, hover, etc.) for this button. :writing_hand: